components/app-core/frontend/i18n/en/common.json

```diff
@@ -28,7 +28,8 @@
   },
   "navbar": {
     "title": "@:product.name",
-    "logo": "home"
+    "logo": "home",
+    "copyright": "&copy; SUSE 2017"
```

Comment:
I thought about that - I figured the copyright was with SUSE, regardless of the brand. It can still be overridden by another brand.